service/security/shiro/src/main/java/org/eclipse/kapua/service/authentication/credential/shiro/CredentialImpl.java

```diff
@@ -107,7 +107,7 @@ public CredentialImpl(KapuaId scopeId, KapuaId userId, CredentialType credential
         this.userId = (KapuaEid) userId;
         this.credentialType = credentialType;
         this.credentialKey = credentialKey;
-        this.credentialStatus = credentialStatus;
+        this.status = credentialStatus;
```

Comment:
Change `credentialStatus` constructor parameter to `status` to make everything more consistent with the other parameters